code/modules/mob/living/basic/blob_minions/blob_spore.dm

```diff
@@ -0,0 +1,127 @@
+/**
+ * A floating fungus which turns people into zombies and explodes into reagent clouds upon death.
+ */
+/mob/living/basic/blob_spore
```

Comment:
Now, do we want `/mob/living/basic/blob_creature` as a base for spores, zombies, and blobbers?

Reply:
i've often pondered on abusing the define appendage trick to be able to do two things at once

A) be able to have a shared framework between multiple different types of mob
B) keep a clean slate where we can avoid having to implement massive common frameworks between different mob subtypes

so we would do something like the MAPPING DIRECTIONAL HELPERS macros

```dm
#define DECLARE_MOB_BLOB(path)\
 ##path{\
  faction = FACTION_BLOB; \
  mob_biotypes = BLOB_BIOTYPE; \
 } // etc as more things are shared
```

there are a few obvious drawbacks
A) no common procs without elements or middleman or stuff defined on `/mob/living/basic` and copy-pasting the same thing 
B) might not be a familar pattern to coders and they might overwrite it in weird ways, and there's no easy way to opt-out of a certain variable unlike just simply changing it piecemeal on the subtype
C) etc.

i'm very tired and this doesn't make any sense especially for this current thing. i just figured i'd mention this thought i was sitting on for a few months now here in case you find use for it in future refactors